test/abstractarray.jl

```diff
@@ -110,6 +124,7 @@ function test_scalar_indexing{T}(::Type{T}, shape)
             for i1 = 1:size(B, 1)
                 i += 1
                 @test A[i1,i2,i3] == B[i1,i2,i3] == i
+                # @test
```

Comment:
No worries. Don't rebase until the tests are done, because this won't affect them.